src/lib/payloadCache/PayloadCache.js

```diff
@@ -0,0 +1,46 @@
+const RedisCache = require('@mojaloop/central-services-shared/src/util/redis/redisCache')
```

Comment:
The new files should contain the latest "License" header at the beginning of the file.